x/oracle/client/cli/txUpgradeOracleProposal.go

```diff
@@ -0,0 +1,89 @@
+package cli
+
+import (
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	govcli "github.com/cosmos/cosmos-sdk/x/gov/client/cli"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+	"github.com/medibloc/panacea-core/v2/x/oracle/types"
+	"github.com/spf13/cobra"
+)
+
+const (
+	FlagUpgradeUniqueID = "upgrade-unique-id"
+	FlagUpgradeHeight   = "upgrade-height"
+)
+
+func CmdUpgradeOracleProposal() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "oracle-upgrade (--upgrade-unique-id [uniqueID]) (--upgrade-height [height]) [flags]",
+		Args:  cobra.ExactArgs(0),
+		Short: "Submit a oracle upgrade proposal",
+		Long: "Submit a oracle upgrade along with an initial deposit.\n + " +
+			"You must enter the uniqueID and block height of the Oracle to perform the upgrade.",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+			from := clientCtx.GetFromAddress()
+
+			depositStr, err := cmd.Flags().GetString(govcli.FlagDeposit)
+			if err != nil {
+				return err
+			}
+
+			deposit, err := sdk.ParseCoinsNormalized(depositStr)
+			if err != nil {
+				return err
+			}
+
+			content, err := parseArgsToContent(cmd)
+			if err != nil {
+				return err
+			}
+
+			msg, err := govtypes.NewMsgSubmitProposal(content, deposit, from)
+			if err != nil {
+				return err
+			}
+
+			return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), msg)
+		},
+	}
+
+	cmd.Flags().String(govcli.FlagTitle, "", "title of proposal")
+	cmd.Flags().String(govcli.FlagDescription, "", "description of proposal")
+	cmd.Flags().String(govcli.FlagDeposit, "", "deposit of proposal")
+	cmd.Flags().Int64(FlagUpgradeHeight, 0, "The height at which the upgrade must happen")
+	cmd.Flags().String(FlagUpgradeUniqueID, "", "Oracle's uniqueID to be upgraded")
```

Comment:
If this two flags are required, why don't we mark these two flags as requirement?

Reply:
Thanks. I added it.(https://github.com/medibloc/panacea-core/pull/454/commits/d45982338677ec29cb075c64c43aa842c350f3e9)